### ISSUE TITLE
Items losing their percentage when editing through the GUI, as well as commands reverting to items.

### ISSUE THREAD

**Issue by Bumbullbee** [ConnorLinfoot/CratesPlus#265]
**Please use the issue template below. Issues not using the templates will likely be closed! If you are suggesting a feature you do not need to follow the template.**

If you're having config issues (such as it resetting) first make sure it's a valid YAML file using a [parser](http://yaml-online-parser.appspot.com).

From version 4.2 Snapshot 1 there is a new '/cratesplus debug' command. Running this will give you a single link with all the data I need to help you. If you submit that you won't need to submit your plugin list, version, or any files.

### Plugin list:
WorldEdit, SavageDeathCompass, Ram_Monitor, Do_Math, GroupManager, PacketListenerApi, MapManager, Thirst, ProtocolLib, WorldGuard, CraftElytra, LibsDisguises, LoreAttributesRecoded, PhysicsToGo, MOTDgdCommandAd, CustomTime, mcMMO, Elytra, MoreHearts, Vault, Lores, BoilingCauldrons, InfiniteBlocks, ArmorWeight, MoneyMobs, GiantTrees, SavageDeathChest, Essentials, HolographicDisplays, AdMaps, EssentialsGeoIP, EssentialsChat, TreeAssist, ServerSigns, SuperVanish, EssentialsSpawn, EnjinMinecraftPlugin, CratesPlus, CSGO-CratesPlus

### Actions to reproduce (if known):
After editing the config to have all items in a certain chest contain a percentage open the item editor through the GUI (/crate settings). After doing so and going back into the config the percentages are lost.

Also after doing this process any commands that were part of the config will be reverted to items, but retain their name. 

### CratesPlus Version:
4.1.1

### CratesPlus config.yml
http://pastebin.com/wEwSTPMV

### CratesPlus data.yml
http://pastebin.com/HcXR4WWH

**Comment by ConnorLinfoot** [comment]
Yeah I'm aware of this. Really at the moment if you edit the config file you shouldn't use the GUI, if you use the GUI you shouldn't edit the config file.

Of course I'd prefer it to keep permissions, commands, percentages etc but I think that'll be something for v5.0.
